### PR TITLE
Implement getRegionalValues in util and refactor $.ig.formatter function

### PR DESCRIPTION
**Pull request by dkamburov**

### CODE REVIEW

File: src/js/modules/infragistics.util.jquery.js [hunk 1]
```diff
@@ -122,236 +122,277 @@
 	//checkbox markup classes can be changed
 	//and setting them in a a box can be done by the predefined classes "ui-state-default ui-corner-all ui-igcheckbox-small"
 	$.ig.checkboxMarkupClasses = "";
+	$.ig.getRegionalOptions = function (reg) {
```
Comment (kdinev) [inline src/js/modules/infragistics.util.jquery.js]: Don't these two belong to the base widget, same as `getLocaleValue`, or only `getRegionalValue`?

Comment (dkamburov) [inline src/js/modules/infragistics.util.jquery.js]: The util is not a widget, so I cannot extend the base widget here.

Comment (kdinev) [inline src/js/modules/infragistics.util.jquery.js]: My question is why `getRegionalValue` belongs to util. It's strange to me that there is a function which takes the object it extracts from as a parameter and is on a utility level.

Can you show the examples of how, and from where this method is used, apart from the formatter? Also, how is this integrated in the editors?

Comment (dkamburov) [inline src/js/modules/infragistics.util.jquery.js]: Let's have a meeting to discuss this.

**Pull request merged**